lib/constants/http_mock_response.dart

```diff
@@ -23,33 +23,35 @@ class HttpMockResponse {
       id: 1,
       creator: "creator",
       recipient: "recipient",
-      quantity: "1000000.0000 SEEDS",
+      quantity: 1000000.00,
```

Comment:
real response still returns balances in format "1000000.0000 SEEDS"
this format should be used everywhere for consistency
but it can be displayed with two decimals when it has zeros for better design

Reply:
we can just split the string into 2 - the number part and the SEEDS part - and then parse the first one as number and get (floating point) number.

Then format for display any way we like

This format is fixed as a response from the blockchain. Of course a regexp can be used but I feel it's not necessary as it's trivial to parse this and things can go wrong with regex.

Reply:
> This is cool but throughout the app numbers should be formatted with no more than 2 trailing zeroes.
> We can use the same formatting and display everywhere
> 
> Examples
> 
> 7.00 SEEDS
> 7.00
> 7.10 SEEDS
> 7.10
> 
> 500.0001 SEEDS (note: Not a trailing zero)
> 500.0001
> 
> 27.049 SEEDS 
> 27.049
> 
> 5.50 SEEDS
> 5.00 SEEDS
> 5.55 SEEDS
> 5.05 SEEDS
> 5.005 SEEDS
> 5.0005 SEEDS
>